jupyterhub_home_nfs/tests.py

```diff
@@ -1,14 +1,16 @@
 import os
```

Comment:
Move this into a 'tests/' folder one level up, as we don't need to ship this as part of the package

Reply:
done!